### ISSUE TITLE
remove external javascript compare tool?

### ISSUE THREAD

**Issue by mmikeww** [mmikeww/AHK-v2-script-converter#36]
could use this all AHK solution:

https://www.autohotkey.com/boards/viewtopic.php?f=6&t=59029

**Comment by dmtr99** [comment]
There are v1 examples of rich edit textbox with line numbers
I have started an attempt to convert the richedit Class.
https://www.autohotkey.com/boards/viewtopic.php?f=82&t=94397

**Comment by mmikeww** [comment]
I'm not sure how necessary it is to move away from the javascript diff tool, it seems to work fine

**Comment by dmtr99** [comment]
True, but I was kind of interested in how richedit works, but could not find a working V2 Version.

I was missing some some custom functionalities so I created my own Gui script to quickly convert and write scripts (testFiles.ahk). I will probably rename it and mark is as a possible tester tool. I use it a lot. I also added a button to compare the scripts in VS code, as it is indeed silly to spend a lot of time to something that already exists. 
Helpfull features are:

 - Looking up the ahk help files of the different versions
 - Testing (view V1, converted V2 and Expected V2 Scripts)
 - Creating tests. 
 - Converting selected text (XButton1)
 - Running the v1 and v2 scripts quickly
 - Closing the scripts
 - Compare with VS code (if installed)
 - Displaying the hidden Symbols (I cheat and just replace every hidden simbol :D )
 
In the end, it does not affect the main ConvertFuncs.ahk function.

**Comment by mmikeww** [comment]
I looked at your personal repository, and seems youre doing a ton of work. I also see that you're being much more disciplined to keep each individual commit limited to smaller distinct changes. This is a good practice. Do you want direct write access to this repository, and then you can push your changes directly here?

**Comment by dmtr99** [comment]
You can give it to me if you want, then I will just upload my changes in your repository.

**Comment by mmikeww** [comment]
ok i think i've invited you

**Comment by dmtr99** [comment]
As far as I can see, I do not have write access
I have made a minor pull request :D

**Comment by mmikeww** [comment]
Here's what it shows me:

![image](https://user-images.githubusercontent.com/1900684/132420578-821d0cc4-8b15-4b5d-a354-4dbb485c90f7.png)
